### ISSUE TITLE
[Travis-CI] ERROR: Failed requirement 'mysql-connector-c/6.1.11@bincrafters/stable' from 'PROJECT'

### ISSUE THREAD

**Issue by tt4g** [tt4g/conan-soci#7]
Failed: requirement 'mysql-connector-c/6.1.11@bincrafters/stable'.

Travis CI log.
```
ERROR: Failed requirement 'mysql-connector-c/6.1.11@bincrafters/stable' from 'PROJECT'
```

**Comment by tt4g** [comment]
Found environment variable `CONAN_REMOTES`.
> Use the argument remotes or environment variable CONAN_REMOTES to configure additional repositories containing needed requirements.
>> https://github.com/conan-io/conan-package-tools

**Comment by tt4g** [comment]
Fixes e5e062c6969a465a3b57c2995b3b16b1ce1f666d
